tooling/lib/releases_diff.rb

```diff
@@ -30,26 +50,29 @@ def self.git_root()
     end
 
     # Gets the directory that contains all the final releases downloaded for SCF
-    def current_final_releases_work_dir()
-        current_manifest_path = File.join(ReleasesDiff.git_root, @manifest_path)
-        File.join(File.expand_path('../', current_manifest_path), '.final_releases')
+    # Note FISSILE_FINAL_RELEASES_DIR in `.envrc`.
+    def final_releases_work_dir()
+        File.join(File.expand_path('../', @current_src_path), '.final_releases')
```

Comment:
Maybe `return ENV['FISSILE_FINAL_RELEASES_DIR'] if ENV['FISSILE_FINAL_RELEASES_DIR']` (or something to that effect)?

Reply:
If it will work, sure.

Reply:
Done with commit c92adf2c720739e0e45b885d7ace25e7b88f75ad